### PR TITLE
Add may roadmap

### PR DESCRIPTION
**Pull request by adamltyson**
This PR adds a May 2024 high-level roadmap for all of BrainGlobe. The aim is to communicate plans to users and contributors, and also to set some goals.

I've requested reviews from lots of people as this roadmap touches on work you're all doing. Please suggest any changes, particularly about time frames for your work.

Also, is there anything else we should add?

N.B. I thought about separating the retro from the roadmap, but I thought I would simplify the structure as much as possible to encourage us to do this in the future.

### CODE REVIEW

**Review by willGraham01 — APPROVED** [review]

**Review by niksirbi — APPROVED** [review]

**Review by K-Meech — APPROVED** [review]

File: docs/source/community/roadmaps/may-2024.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# May 2024
+
+The [November 2023 roadmap](november-2023) was our first. This document looks back on the last six months of work 
+and outlines plans for the next year.
+
+## Current status
+
+### Progress since the last roadmap
+The past few months have focussed mostly on improving the maintainability of BrainGlobe as a whole, and making it easier
```
[the remark this answers is not in the document]
Comment (adamltyson) [reply docs/source/community/roadmaps/may-2024.md:9]: Technically "focussed" is correct (British) English, but I'll change it as focused is more common elsewhere.

[the remark this answers is not in the document]
Comment (adamltyson) [reply docs/source/community/roadmaps/may-2024.md]: [Yup](https://en.wikipedia.org/wiki/American_and_British_English_spelling_differences)

File: docs/source/community/roadmaps/may-2024.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* We've released `brainrender-napari` to allow for visualisation of atlas-registered data in napari 
+(alongside the existing `brainrender`).
+
+### Issues
+* It is not yet possible to carry out all BrainGlobe functionality via the GUI (although we are 90% of the way there).
+* Documentation for some tools is sparse
+* Docstrings are often missing, incomplete or not standardised
+* We have very few API docs
+* Test coverage is variable (some repos are in excellent shape, others not so much!)
+* cellfinder uses TensorFlow, and so we're limited to what Python versions we can support for the metapackage)
```
Comment (sfmig) [inline docs/source/community/roadmaps/may-2024.md:35]:
```suggestion
* cellfinder uses TensorFlow, and so we're limited to what Python versions we can support for the metapackage
```

**Review by sfmig — APPROVED** [review]
I like it, very clear

**Review by IgorTatarnikov — APPROVED** [review]
Looks good to me!

File: docs/source/community/roadmaps/may-2024.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+* `imio` has been brought into the `brainglobe-utils` package, to reduce the number of packages we maintain. 
+* `brainrender` has been updated so that it now works with the newest versions of Python.
+* A new `LineActor` has been added to `brainrender`.
+* `bgheatmap` has been updated so that it now works with the newest versions of Python and now released as `brainglobe-heatmap`.
+* `brainglobe` version 1 has been released. This new "metapackage" allows one line installation of all BrainGlobe tools.
+* We've fixed a lot of small bugs and hugely reduced the number of open issues. 
+* We've standardised the tooling for all repos to simplify develpoment and maintenance.
+* We've released `brainrender-napari` to allow for visualisation of atlas-registered data in napari 
+(alongside the existing `brainrender`).
+
```
Comment (alessandrofelder) [inline docs/source/community/roadmaps/may-2024.md:28]:
```suggestion
* We've fixed a number of scaling and rotation issues with some BrainGlobe atlases.
```

File: docs/source/community/roadmaps/may-2024.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+to develop new tools in the future. Some developments include (for more details on these, see the [blog](https://brainglobe.info/blog/)):
+* The `brainreg` napari plugin is now bundled with `brainreg` to simplify maintenance.
+* `brainreg-segment` is now called `brainglobe-segmentation` to reflect it's future compatibility with other registration tools.
+* The old `cellfinder` command-line tool is now called `brainmapper` and is part of `brainglobe-workflows`. This is part of
+work to separate the core BrainGlobe software from convenience tools ("workflows") developed for specific applications.
+* The old `cellfinder-core` and `cellfinder-napari` tools have been merged, and are now known just as `cellfinder`. 
+This is to simplify maintenance and reduce confusion (we used to have three cellfinders!).
+* `bg-space` is now known as `brainglobe-space` to standardise naming.
+* `bg-atlasapi` and `bg-atlasgen` have been merged and are now known as `brainglobe-atlasapi` to standardise naming.
+* `imio` has been brought into the `brainglobe-utils` package, to reduce the number of packages we maintain.
```
Comment (alessandrofelder) [inline docs/source/community/roadmaps/may-2024.md:19]:
```suggestion
* `imio` has been brought into the `brainglobe-utils` package, to reduce the number of packages we maintain. We've increased the robustness of `brainglobe-utils` by adding lots of automated tests.
```

**Review by alessandrofelder — APPROVED** [review]
Some minor suggestions

File: docs/source/community/roadmaps/may-2024.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+* `imio` has been brought into the `brainglobe-utils` package, to reduce the number of packages we maintain. We've increased the robustness of `brainglobe-utils` by adding lots of automated tests.
+* `brainrender` has been updated so that it now works with the newest versions of Python.
+* A new `LineActor` has been added to `brainrender`.
+* `bgheatmap` has been updated so that it now works with the newest versions of Python and now released as `brainglobe-heatmap`.
+* `brainglobe` version 1 has been released. This new "metapackage" allows one line installation of all BrainGlobe tools.
+* We've fixed a lot of small bugs and hugely reduced the number of open issues. 
+* We've standardised the tooling for all repos to simplify develpoment and maintenance.
+* We've released `brainrender-napari` to allow for visualisation of atlas-registered data in napari 
+(alongside the existing `brainrender`).
+* We've fixed a number of scaling and rotation issues with some BrainGlobe atlases.
```
Comment (adamltyson) [inline docs/source/community/roadmaps/may-2024.md:28]:
```suggestion
* We've fixed a number of scaling and rotation issues with some BrainGlobe atlases.

```

**Pull request merged**